trillian/ctfe/cert_checker_test.go

```diff
@@ -320,6 +321,77 @@ func TestNotAfterRange(t *testing.T) {
 	}
 }
 
+func TestRejectExpiredNonExpired(t *testing.T) {
+	fakeCARoots := NewPEMCertPool()
+	if !fakeCARoots.AppendCertsFromPEM([]byte(testonly.FakeCACertPEM)) {
+		t.Fatal("failed to load fake root")
+	}
+	chain := pemsToDERChain(t, []string{testonly.LeafSignedByFakeIntermediateCertPEM, testonly.FakeIntermediateCertPEM})
```

Comment:
Be nice to add a comment reminding the reader of the `NotAfter` date of this leaf.

Reply:
Apparently, not only the leaf validity range is taken into consideration. The `FakeCACertPEM` turned out to have a shorter period, and restricting the date in "reject-expired-pass" test.

Reply:
Done.